src/evict/evict_lru.c

```diff
@@ -160,14 +160,16 @@ __evict_server(void *arg)
 	WT_DECL_RET;
 	WT_EVICT_WORKER *worker;
 	WT_SESSION_IMPL *session;
+	uint64_t laps;
 
 	session = arg;
 	conn = S2C(session);
 	cache = conn->cache;
+	laps = 0;
```

Comment:
This appears to be a constant -- the local variable isn't updated by `__evict_pass`.  What's the idea of counting laps here?